### PR TITLE
mdv: init at 1.6.3

### PR DESCRIPTION
**Pull request by Synthetica9**
###### Motivation for this change

`mvd` wasn't in packages.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -20204,6 +20204,26 @@ EOF
...
+      sha256 = "1x4d1gyvmdqh2zpdc415ak10d4yfli04pw4mxwfv2jnjk94d1y0k";
+    };
+
+    propagatedBuildInputs = with self; [ markdown pygments pyyaml docopt ];
+
+    meta = {
+      homepage = https://github.com/axiros/terminal_markdown_viewer;
+      description = "Terminal Markdown Viewer";
+      license = licenses.bsd;
+      maintainers = with maintainers; [ synthetica ];
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: You need to add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -20204,6 +20204,26 @@ EOF
 
   pypandoc = callPackage ../development/python-modules/pypandoc { };
 
+  mdv = buildPythonPackage rec {
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: Please use `callPackage`.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -20204,6 +20204,26 @@ EOF
...
+    pname = "mdv";
+    name = "${pname}-${version}";
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: Leave this line out.

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -20204,6 +20204,26 @@ EOF
...
+    version = "1.6.3";
+
+    src = pkgs.fetchurl {
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]: `fetchPypi`

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please prefix your commit message with `pythonPackages.`

**Comment by Synthetica9** [comment]
@dotlambda Bump

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Please provide the correct `checkPhase`.

Also, note https://github.com/axiros/terminal_markdown_viewer#manual-install-requirements (`disabled = isPy3k`).

**Comment by aanderse** [comment]
@Synthetica9 are you able to continue with this PR?

**Comment by FRidh** [comment]
Closing due to inactivity.
Nothing happening so closing.

**Pull request closed without merging**